packages/fx-core/package.json

```diff
@@ -30,7 +30,7 @@
     "test:core-tools": "nyc mocha \"tests/core/tools.test.ts\"",
     "test:apiv2": "nyc mocha \"tests/plugins/resource/apiv2/*.test.ts\"",
     "test:solutionv3": "nyc mocha \"tests/plugins/solutionv3/*.test.ts\"",
-    "test:env-checker": "nyc mocha \"tests/common/deps-checker/*.test.ts\"",
+    "test:env-checker": "npx mocha \"tests/common/deps-checker/**/*.test.ts\"",
```

Comment:
why this change?

Reply:
nyc will terminate IT cases by coverage check issue. You can see this [job](https://github.com/OfficeDev/TeamsFx/runs/4614820710?check_suite_focus=true) to learn more about.